app/services/translation_service/api/translations.rb

```diff
@@ -0,0 +1,66 @@
+module TranslationService::API
+
+  class Translations
+
+    TranslationStore = TranslationService::Store::Translation
+
+
+    ## GET /translations/:community_id/
+    def get(community_id, request_params = {})
+      begin
+        params = TranslationService::DataTypes::Translation
```

Comment:
Good question!
Personally, I felt that this (DataTypes) is a good way to test the structure of parameters. 
But how to  do it - suggestions are welcomed.